### DISCUSSION TITLE
[HELP] How to install extension on PostgreSQL 13 Ubuntu 20.04.6?

### DISCUSSION THREAD

**Discussion by Kladislav** [pgroonga/pgroonga#471] [Q&A] [answered]
There is no packages in apt and i cant find any info in docs, please help

**Comment by kou** [comment]
How did you install PostgreSQL 13 on Ubuntu 20.04?
Ubuntu 20.04 ships only PostgreSQL 12.

**Reply by askdkc** [reply]
Adding PostgreSQL apt repository does the trick: https://www.postgresql.org/download/linux/ubuntu/

FYI: https://blog.mindmeldwithminesh.com/how-to-install-postgresql-13-on-ubuntu-20-04-18-04-93f550f78cce

**Reply by Kladislav** [reply]
true, installed a long time ago

**Reply by askdkc** [reply]
@Kladislav 

I think you should install PostgreSQL 14 then you might be able to do

```bash
sudo apt install -y -V postgresql-14-pgroonga
```

**Reply by kou** [reply, marked as answer]
You can use https://pgroonga.github.io/install/ubuntu.html#install-for-official-postgresql for it.
